modules/internal/NetworkAtomics.chpl

```diff
@@ -340,35 +340,35 @@ module NetworkAtomics {
     return lhs;
   }
 
-  inline proc =(ref a:RAtomicBool, const b:RAtomicBool) {
+  inline operator =(ref a:RAtomicBool, const b:RAtomicBool) {
```

Comment:
see my comment for Atomics.chpl - I would think these should be operator methods

Reply:
It looks like the ones in Atomics.chpl got changed, but not these ones in NetworkAtomics.hcpl